bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/filters/ValueFilterAdapter.java

```diff
@@ -65,27 +63,23 @@ public FilterSupportStatus isFilterSupported(
 
   private RowFilter adaptBinaryComparator(
       CompareOp compareOp, BinaryComparator comparator) throws IOException {
-    byte[] comparatorValue = comparator.getValue();
-    ByteArrayOutputStream baos =
-        new ByteArrayOutputStream(comparatorValue.length * 2);
-    ReaderExpressionHelper.writeQuotedRegularExpression(baos, comparatorValue);
```

Comment:
writeQuotedRegularExpression() is used to escape special characters. I wonder why this is not needed now. Notice that writeQuotedRegularExpression() is used in other places too. If this is a problem, we need to remove it all together.